lib/event-source.js

```diff
@@ -0,0 +1,79 @@
+var mongojs = require('mongojs');
+var inflect = require('i')();
+var _ = require('lodash');
```

Comment:
tinysse was removed, however the heartbeat mechanism is gone along with it 
which is required for keeping the connection open across some proxies